cpu/cc2538/periph/uart.c

```diff
@@ -106,49 +103,6 @@ static void reset(cc2538_uart_t *u)
 }
 /*---------------------------------------------------------------------------*/
 
-#if UART_0_EN
-void UART_0_ISR(void)
-{
-    uint_fast16_t mis;
-
-    /* Latch the Masked Interrupt Status and clear any active flags */
-    mis = UART_0_DEV->cc2538_uart_mis.MIS;
-    UART_0_DEV->ICR = mis;
-
-    while (UART_0_DEV->cc2538_uart_fr.FRbits.RXFE == 0) {
-        uart_config[0].rx_cb(uart_config[0].arg, UART_0_DEV->DR);
-    }
-
-    if (mis & (OEMIS | BEMIS | FEMIS)) {
-        /* ISR triggered due to some error condition */
-        reset(UART_0_DEV);
-    }
-
-    cortexm_isr_end();
-}
-#endif /* UART_0_EN */
-
-#if UART_1_EN
-void UART_1_ISR(void)
-{
-    uint_fast16_t mis;
-
-    /* Latch the Masked Interrupt Status and clear any active flags */
-    mis = UART_1_DEV->cc2538_uart_mis.MIS;
-    UART_1_DEV->ICR = mis;
-
-    while (UART_1_DEV->cc2538_uart_fr.FRbits.RXFE == 0) {
-        uart_config[1].rx_cb(uart_config[1].arg, UART_1_DEV->DR);
-    }
-
-    if (mis & (OEMIS | BEMIS | FEMIS)) {
-        /* ISR triggered due to some error condition */
-        reset(UART_1_DEV);
-    }
-
-    cortexm_isr_end();
-}
-#endif /* UART_1_EN */
 
 static int init_base(uart_t uart, uint32_t baudrate);
```

Comment:
please dropt the `init_base()` function and integrate it directly into `uart_init()`. This is a leftover from some former refactoring and should be resolved.